src/meshdb/settings.py

```diff
@@ -36,6 +36,26 @@
 
 USE_X_FORWARDED_HOST = True
 
+MESHDB_ENVIRONMENT = os.environ.get("MESHDB_ENVIRONMENT", "")
+if MESHDB_ENVIRONMENT == "":
+    # Local or grand
+    LOS_URL = "https://los.grandsvc.mesh.nycmesh.net"
+    MAP_URL = "https://map.grandsvc.mesh.nycmesh.net"
+    PG_ADMIN_URL = "/pgadmin/"
+    FORMS_URL = "https://forms.grandsvc.mesh.nycmesh.net"
```

Comment:
Hmm there is overlap here in functionality with several values in `.env`:
- `SITE_BASE_URL`
- `ADMIN_MAP_BASE_URL`
- `PELIAS_ADDRESS_PARSER_URL`
- `UISP_URL`

Should we consolidate these other URLs to this style? I'm worried about having two different systems and it not being clear which one should be used for a given URL. It seems like you only changed this for the links shown to users (and the ones from `.env` are values used internally). I think having that as the standard is fine but I do wish there was a way to make it a bit more clear for newer folks.

Adding these `.env` URLs here would also introduce the issue of local testing overrides causing a `git diff` (a pet peeve of mine). Perhaps they could be included here and the `.env` values could act as an override?

Reply:
Also perhaps this should default to dev rather than grandbox, since I thought we were putting grandbox out to pasture

Reply:
Done, please have a look

Reply:
So many clicks